packages/hooks/src/useNotificationEvent.ts

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { useMutation, useInvalidateQuery } from '@deriv/api';
+import { TSocketRequestPayload } from '@deriv/api/types';
+
+type TNotificationEventPayload = TSocketRequestPayload<'notification_event'>['payload'];
+
+/**
+ * Hook to send notification event to the server
+ * @name useNotificationEvent
+ * @returns response, mutation function and other properties from useRequest hook
+ */
+const useNotificationEvent = () => {
+    const invalidate = useInvalidateQuery();
+    const { data, mutate, ...rest } = useMutation('notification_event', {
+        onSuccess: () => {
+            invalidate('notification_event');
```

Comment:
invalidateQueries is for invalidating useQuery data, why do we need to invalidate here?

Reply:
I think its not required here. But invalidateQueries can be used on mutation if we want to retrigger getters or have a side effect on another API

Reply:
Yess, its fine to put it if we have `useQuery` with `notification_event` key